src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Services/ContentApiClientTests.cs

```diff
@@ -0,0 +1,130 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using RichardSzalay.MockHttp;
+using System.Net;
+using System.Text.Json;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests.Services;
+
+public class ContentApiClientTests
+{
+    private readonly MockHttpMessageHandler _mockHttp;
+    private readonly HttpClient _client;
+    private readonly Mock<ILogger<ContentApiClient>> _logger;
+    private readonly ContentApiClient _contentApiClient;
+
+    public ContentApiClientTests()
+    {
+        _mockHttp = new MockHttpMessageHandler();
+        _client = _mockHttp.ToHttpClient();
+        _client.BaseAddress = new Uri("http://localhost/");
+        _logger = new Mock<ILogger<ContentApiClient>>();
+        _contentApiClient = new ContentApiClient(_logger.Object, _client);
+    }
+
+    [Fact]
+    public async Task ListPublications_HttpClientRespondsWithBadRequest_ReturnsBadRequestObjectResultAndLogsError()
+    {
+        _mockHttp.Expect(HttpMethod.Post, "http://localhost/api/publications")
+            .Respond(HttpStatusCode.BadRequest, new StringContent("test message"));
+
+        var response = await _contentApiClient.ListPublications(
+            It.IsAny<int>(), 
+            It.IsAny<int>(), 
+            It.IsAny<string>(),
+            It.IsAny<IEnumerable<Guid>>());
+
+        _mockHttp.VerifyNoOutstandingExpectation();
+
+        _logger.Verify(logger => logger.Log(
+                It.Is<LogLevel>(logLevel => logLevel == LogLevel.Error),
+                It.Is<EventId>(eventId => eventId.Id == 0),
+                It.Is<It.IsAnyType>((@object, @type) =>
+                    @object.ToString() == 
+                        $"""
+                         Failed to retrieve publications with status code: {HttpStatusCode.BadRequest}. Message:
+                         test message
+                         """.TrimIndent()),
+                It.IsAny<Exception>(),
+                It.IsAny<Func<It.IsAnyType, Exception?, string>>()),
+            Times.Once);
+
+        var left = response.AssertLeft();
+        Assert.IsType<BadRequestObjectResult>(left);
+    }
+
+    [Theory]
+    [InlineData(HttpStatusCode.RequestTimeout)]
+    public async Task
+        ListPublications_HttpClientRespondsWithUnsuccessfulStatusCode_ThrowsHttpRequestExceptionAndLogsError(
```

Comment:
Similar to above - could make this less verbose i.e. `ListPublications_HttpClientFailureStatusCode_ThrowsException`

Not sure there's much value in providing the specific exception in the method, in case that ever changes.